modules/tinymce/src/core/main/ts/api/util/Tools.ts

```diff
@@ -248,24 +248,18 @@ const create = function (s, p, root?) {
   });
 };
 
-const extend = function (obj, ext, ...x: any[]) {
-  let i, l, name;
-  const args = arguments;
-  let value;
-
-  for (i = 1, l = args.length; i < l; i++) {
-    ext = args[i];
-    for (name in ext) {
+const extend = function (obj, ...exts: any[]) {
+  for (let i = 0; i < exts.length; i++) {
+    const ext = exts[i];
+    for (let name in ext) {
       if (ext.hasOwnProperty(name)) {
-        value = ext[name];
-
+        const value = ext[name];
```

Comment:
Rewrote to avoid using `arguments`

Reply:
I wonder if this could be changed to just do `const extend = Merger.merge`. Better left for another time anyways, so just mentioning it to get thoughts more than anything 🤷‍♀